src/components/views/Registration.js

```diff
@@ -32,16 +34,27 @@ FormField.propTypes = {
   onChange: PropTypes.func
 };
 
+const genderOptions = [
+    {value: 'MALE', label: 'Male'},
+    {value:  'FEMALE', label: 'Female'},
+    {value: 'OTHER', label: 'Other'}
```

Comment:
Do we really want Other? I see why you wouldn't want to be exclusive, but the sex will be mainly used for preferences regarding playing with people from that gender. What can one expect if he/she selects "other" in their preference setting?

Reply:
I would leave it as is.. yes we might not be able to imagine what other means but it does not require much effort to implement and we will be more inclusive..